engines/standard_tasks/app/mailers/standard_tasks/paper_reviewer_mailer.rb

```diff
@@ -0,0 +1,18 @@
+module StandardTasks
+  class PaperReviewerMailer < ActionMailer::Base
+    include Rails.application.routes.url_helpers
+
+    default from: ENV.fetch('FROM_EMAIL')
+
+    def notify_invited(invitation_id:)
+      invitation = Invitation.find(invitation_id)
+      @invitee = invitation.invitee
+      @paper = invitation.paper
+
+      mail({
+        to: invitation.email,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.